packages/website/ts/pages/jobs/open_positions.tsx

```diff
@@ -0,0 +1,182 @@
+import * as _ from 'lodash';
+import CircularProgress from 'material-ui/CircularProgress';
+import { Table, TableBody, TableHeader, TableHeaderColumn, TableRow, TableRowColumn } from 'material-ui/Table';
+import * as React from 'react';
+
+import { Retry } from 'ts/components/ui/retry';
+import { HeaderItem } from 'ts/pages/jobs/list/header_item';
+import { ListItem } from 'ts/pages/jobs/list/list_item';
+import { colors } from 'ts/style/colors';
+import { styled } from 'ts/style/theme';
+import { ScreenWidths, WebsiteBackendJobInfo } from 'ts/types';
+import { backendClient } from 'ts/utils/backend_client';
+
+const labelStyle = { fontFamily: 'Roboto Mono', fontSize: 18 };
+const HEADER_TEXT = 'Open Positions';
+const LIST_ITEM_MIN_HEIGHT = 80;
+
+export interface OpenPositionsProps {
+    hash: string;
+    screenWidth: ScreenWidths;
+}
+export interface OpenPositionsState {
+    jobInfos?: WebsiteBackendJobInfo[];
+    error?: Error;
+}
+
+export class OpenPositions extends React.Component<OpenPositionsProps, OpenPositionsState> {
+    private _isUnmounted: boolean;
+    constructor(props: OpenPositionsProps) {
+        super(props);
+        this._isUnmounted = false;
```

Comment:
Why this `_isUnmounted` stuff? https://reactjs.org/blog/2015/12/16/ismounted-antipattern.html
Does it work to just fetch in `componentDidMount` ?

Reply:
It seems that tracking an internal variable is something that they suggest in that link. I believe the issue is if the fetch takes a long time to resolve and then component has unmounted already

Reply:
yea I guess you either do this or implement the cancelling thing they recommend. I think the new React API handles this a bit better.